CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,200 @@
+* Sustained disruption of community spaces or events.
+
+## Consequences of unacceptable behaviour
+
+Unacceptable behaviour from any community member, including anyone with
+decision-making authority, will not be tolerated. Anyone asked to stop
+unacceptable behaviour is expected to comply immediately.
+
+If a community member engages in unacceptable behaviour, the community
+organizers may take any action they deem appropriate, by agreement with, and
+subject to correction and reversal by, the lead organizer. These include, but 
+are not limited to:
+
+* A verbal warning (either in private or in public).
+* A temporary ban of any length, with or without right of appeal, from any or
+  all community spaces.
+* A permanent ban, with or without right of appeal, from any or all community
+  spaces.
+* Temporary revocation of community organizer status for any length of time, 
+  with or without right of appeal.
+* Permanent revocation of community organizer status with or without right of 
+  appeal.
+* Removal of contributions to Pext from any Pext-associated repositories, with
+  or without right of appeal.
+* Notification of law enforcement, or other appropriate authorities, along with
+  any evidence of wrongdoing provided to the community organizers.
+
+## Weapons policy
+
+No weapons are allowed at Pext community events, community spaces, or in other
+spaces covered by the scope of this Code of Conduct. Weapons include, but are
+not limited to, guns, explosives (including fireworks without prior approval by
+a community organizer), and large knives such as those used for hunting or
+display, as well as any other item used for the purpose of causing injury and
+harm to others. Anyone seen in the possession of these items will be asked to
+leave immediately, and will only be allowed to return without the weapon.
+Community members are further required to comply with all local laws and
+regulations on this matter.
+
+Repeated violations of this policy are grounds for enforcement actions as if
+against unacceptable behaviour.
+
+## Reporting guidelines
+
+If you are subject to, or witness, unacceptable behaviour, or have any other
+concerns, please notify a community organizer as soon as possible. The current
+list of community organizers is:
+
+* Sylvia van Os <sylvia@hackerchick.me> (lead organizer)
+
+When reporting, please include the following information, to the best of your
+knowledge and ability:
+
+* When and where the unacceptable behaviour occurred.
+* Who engages in unacceptable behaviour.
+* Which unacceptable behaviour(s) happened.
+* Any evidence for your report (such as screencaps, logs, etc)
+* Who (besides yourself) was affected by this behaviour.
+* Whether you want any additional actions taken by the community organizers to 
+  ensure that your safety and well-being remain intact.
+
+Additionally, community organizers can assist local authorities and connect you
```

Comment:
@VenomousDingo You said this word should be "can", right? I still feel "may" seems better because it doesn't seem to feel as much like a promise as "can"? Why do you feel can is better? Am I missing some grammar thing?

Reply:
The use of 'may', to me at least, carries the interpretation 'if we feel like it', which I find to be totally against the spirit of this sentence. 'Can', on the other hand does not. I admit that it carries the interpretation of promising to do it, but not nearly as strongly IMHO. I welcome alternative phrasings, though.

Reply:
How about "may be able to"? It doesn't promise that we can help, but it does imply that we'll try to help if possible. Same for the "will also" a few sentences later, how about "will also try to"?

Reply:
To me, that sounds too watery. It's important that a document like this strikes a sufficiently imperative and competent tone to reassure people that we will act with proper due diligence when responding to unacceptable behaviour and making our community feel safe. Given that this is 2018, which means drive-by sexism and racism is unfortunately very common, and at least one woman in tech feels unsafe enough to _return to her own home_, I think we need to really stake out our position in much stronger language than what you are suggesting.

Reply:
The problem here is that "can" really promises we can help. I haven't ever worked with law enforcement. I don't want to promise I can get it right. Would "will attempt to assist local authorities to their best efforts" work? Because that I will obviously do, without doubt.

Reply:
Sure, I can go with that. I'll make the necessary edits.